.storybook/recipes/CalendarCard/CalendarCard.tsx

```diff
@@ -8,7 +8,7 @@ import type { Variant as TagVariant } from '../../../src/components/Tag/Tag';
 
 export const VARIANTS = ['brand', 'revise', 'success'] as const;
 
-export type Variant = typeof VARIANTS[number];
+export type Variant = (typeof VARIANTS)[number];
```

Comment:
I'm actually not sure why these lint violations aren't reported on next but are reported on my branch, maybe I messed up the deps on my branch 🤔